src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```diff
@@ -23,3 +23,15 @@ admin_merchandise_returns_filter:
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
 #    _legacy_link: AdminReturn:submitFilterorder_return
+
+admin_order_returns_edit:
+    path: /{orderReturnId}/edit
+    methods: [GET, POST]
+    defaults:
+        _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\MerchandiseReturnController::editAction
+        _legacy_controller: AdminReturn
+    #    _legacy_link: AdminReturn:updateorder_return
```

Comment:
could add a todo reminder at the top of yml file, as I suppose these legecay links should be uncommented once the page is fully shipped to production?